_studio/shared/umc/codec/h264_dec/src/umc_h264_va_packer.cpp

```diff
@@ -731,6 +731,7 @@ void PackerVA::PackProcessingInfo(H264DecoderFrameInfo * sliceInfo)
 
     pipelineBuf->surface = m_va->GetSurfaceID(sliceInfo->m_pFrame->m_index); // should filled in packer
     pipelineBuf->additional_outputs = (VASurfaceID*)vpVA->GetCurrentOutputSurface();
+    pipelineBuf->input_color_properties.chroma_sample_location = VA_CHROMA_SITING_HORIZONTAL_LEFT | VA_CHROMA_SITING_VERTICAL_CENTER;
```

Comment:
Please add some comment **in code** why exactly "HORIZONTAL_LEFT |  VERTICAL_CENTER" is set. It's not obvious why this hardcoded value is chosen.

Reply:
> Please add some comment **in code** why exactly "HORIZONTAL_LEFT | VERTICAL_CENTER" is set. It's not obvious why this hardcoded value is chosen.

Upadate code comment:     // To keep output aligned, decode downsampling use this fixed combination of chroma sitting type.